spec/system/assignment_spec.rb

```diff
@@ -57,6 +153,31 @@
       click_link "View"
       check_show_page(name, deadline, description, "Input, Button")
     end
+
+    it "is not able to edit assignment when assignment is closed" do
+      sign_in @mentor
+      @assignment = FactoryBot.create(:assignment, group: @group, status: "closed")
```

Comment:
**Suggestions** Avoid using instance variables in specs, and use let wherever possible https://github.com/rubocop/rspec-style-guide#instance-variables